src/Caching/Hybrid/src/Internal/DefaultHybridCache.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
+
+/// <summary>
+/// The inbuilt ASP.NET implementation of <see cref="HybridCache"/>
+/// </summary>
+internal sealed class DefaultHybridCache : HybridCache
+{
+    private readonly IDistributedCache backendCache;
```

Comment:
`_` prefixed member names. And remove `this.` when referencing them.

Reply:
I almost commented, but I was expecting this to vary by project. 🙃